staging_service/app.py

```diff
@@ -106,6 +118,44 @@ async def bulk_specification(request: web.Request) -> web.json_response:
     return web.HTTPInternalServerError(text=errtext, content_type=_APP_JSON)
 
 
+@routes.post("/write_bulk_specification/")
+async def write_bulk_specification(request: web.Request) -> web.json_response:
+    username = await authorize_request(request)
+    if request.content_type != _APP_JSON:
+        # There should be a way to get aiohttp to handle this but I can't find it
```

Comment:
For example in Jersey you can annotate a method so that it knows it should only accept JSON and therefore there's no need to check the `content-type` header in the method: https://github.com/kbase/auth2/blob/a9939a92b1da8e96fd504be550d607ec0e26b2b8/src/us/kbase/auth2/service/api/Token.java#L84

I was thinking that there should be some way to do a similar thing in aiohttp but I haven't been able to find it if there it one